src/Native/LibTorchSharp/THSTensor.h

```diff
@@ -1317,6 +1317,7 @@ EXPORT_API(Tensor) THSTensor_view_as_complex(const Tensor tensor);
 EXPORT_API(Tensor) THSTensor_view_as_real(const Tensor tensor);
 
 EXPORT_API(Tensor) THSTensor_where(const Tensor condition, const Tensor x, const Tensor y);
+EXPORT_API(void) THSTensor_where_(const Tensor condition, Tensor* (*allocator)(size_t length));
```

Comment:
The pytorch convention is to end in-place operator names with an '_', and I've been using that for .NET and the native methods, too. Maybe something like THSTensor_where_list?